packages/shared-lib/src/input-gateway/deviceTriggerPreviews.ts

```diff
@@ -22,6 +22,17 @@ export interface IWrappedAdLibBase {
 	item: unknown
 }
 
+export interface ShiftRegisterActionArguments {
+	type: 'modifyRegister'
+	register: number
```

Comment:
I think a slightly more descriptive name for this property could be `registerIndex`, it's not a big deal though.